plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/registry/SQLVariablesRegistry.java

```diff
@@ -78,10 +79,14 @@ public static synchronized SQLVariablesRegistry getInstance() {
 
     private void loadVariables() {
         Path configLocation = getConfigLocation();
-
+        if (Files.notExists(configLocation)) {
+            return;
```

Comment:
Maybe this is a good place for logging? like "can't find variables configuration file"?

Reply:
I don't think so because this is a common situation. Most people don't use variables in SQL editor.